docs/ioformats.rst

```diff
@@ -131,3 +130,6 @@ convert to a given PCM subtype and additionally support several compressed forma
 
     # Write out audio as 16bit OGG
     sf.write('stereo_file.ogg', data, samplerate, format='ogg', subtype='vorbis')
+
+
+In general, we recommend using `PySoundFile` for output rather than ``librosa.output.write_wav``, which is mainly provided as a convenience function.
```

Comment:
This sentence is a bit vague. I'm not sure what a "convenience function" is.

Reply:
Fair enough; i'll simplify.